wasi.go

```diff
@@ -53,13 +53,17 @@ func StartWASICommandFromSource(r Runtime, source []byte) (wasm.Module, error) {
 //	module, _ := StartWASICommand(r, decoded)
 //	defer module.Close()
 //
-// Prerequisites of the "Current Unstable ABI" from wasi.ModuleSnapshotPreview1:
-// * "_start" is an exported nullary function and does not export "_initialize"
-// * "memory" is an exported memory.
+// ## "memory" export
+// WASI snapshot-01 requires exporting a memory named "memory", and wazero enforces this as nearly all functions use
+// memory to implement multiple returns. StartWASICommand errs if there is no memory exported as "memory".
+//
+// ## "_start" function export
+// WASI snapshot-01 requires exporting a function named "_start", but wazero does not enforce this. If it is defined,
```

Comment:
nit but important as some embedders will use `_initialize` of WASI reactors e.g. Envoy (though we don't have `InitializeWASIReactor` yet :D)

```suggestion
// WASI snapshot-01 requires exporting a function named "_start" for WASI command, but wazero does not enforce this. If it is defined,
```

Reply:
great point!